tools/rapids-package-name

```diff
@@ -60,6 +67,11 @@ if (( append_pyver == 1 )); then
   pkg_name+="_py${RAPIDS_PY_VERSION//./}"
 fi
 
+# for wheels that bundle shared libraries, add generic pyver
+if (( append_generic_pyver == 1 )); then
+  pkg_name+="_py3"
+fi
```

Comment:
Let's consolidate this with `RAPIDS_PY_WHEEL_PURE` because it does basically the same thing.

"Pure" wheels can be either _just_ Python code (no extension modules unless they're abi3 or similar) or _just_ bundles of shared libraries that have no specific need for a Python minor version. Examples of the former include dask-cudf. Examples of the latter include RAPIDS C++ wheels or CUDA wheels.

Also just to be clear, we don't need a generic Python version string like `_py3` in the artifact name, it is already part of the wheel name contained in the artifact. All that we care is that artifact names are self-consistent and can be identified by human readers.

Reply:
seems like I should just removed this bit completely. Thanks for the simplification.